common/lib/xmodule/xmodule/x_module.py

```diff
@@ -54,6 +54,10 @@
 # the XBlock also implements author_view.
 STUDENT_VIEW = 'student_view'
 
+# This is the view that will be rendered to display the XBlock in the LMS when
+# the user is unauthenticated and the course visibility is public.
```

Comment:
"This is the view that will be rendered to display the XBlock in the LMS for preview. Implementations of this view should assume that a user and user data are not available."